Streamtools crashes when a toEmail block deleted

Deleting a toEmail block causes Streamtools to crash! :(

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x60 pc=0x1e0f2e]

goroutine 30 [running]:
runtime.panic(0x3f2240, 0x9703b9)
    /usr/local/Cellar/go/1.2.1/libexec/src/pkg/runtime/panic.c:266 +0xb6
net/smtp.(*Client).hello(0x0, 0x4dab3, 0x160)
    /usr/local/Cellar/go/1.2.1/libexec/src/pkg/net/smtp/smtp.go:74 +0x1e
net/smtp.(*Client).Quit(0x0, 0x4dab3, 0x4dab3)
    /usr/local/Cellar/go/1.2.1/libexec/src/pkg/net/smtp/smtp.go:343 +0x27
github.com/nytlabs/streamtools/st/library.(*ToEmail).closeClient(0xc2100e3000, 0xc210112000, 0xdf9f58)
    /Users/iain/go/src/github.com/nytlabs/streamtools/st/library/toEmail.go:66 +0x2e
github.com/nytlabs/streamtools/st/library.(*ToEmail).Run(0xc2100e3000)
    /Users/iain/go/src/github.com/nytlabs/streamtools/st/library/toEmail.go:260 +0x2fc
created by github.com/nytlabs/streamtools/st/blocks.BlockRoutine
    /Users/iain/go/src/github.com/nytlabs/streamtools/st/blocks/blocks.go:215 +0xf2
```

--- a/st/library/toEmail.go
+++ b/st/library/toEmail.go
@@ -63,7 +63,11 @@ func (e *ToEmail) initClient() error {
 func (e *ToEmail) closeClient() error {
 	// quit, close and return
 	var err error
-	if err = e.client.Quit(); err != nil {
+	if e.client == nil {
+		return nil
+	}
+	err = e.client.Quit()
+	if err != nil {
 		// quit failed. try a simple close
 		err = e.client.Close()
 	}
@@ -257,7 +261,8 @@ func (e *ToEmail) Run() {
 
 		case <-e.quit:
 			// quit, close and return
-			if err = e.closeClient(); err != nil {
+			err := e.closeClient()
+			if err != nil {
 				e.Error(err.Error())
 			}
 			return

